unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -207,6 +207,11 @@ private GLTFSceneImporter(ILoader externalDataLoader)
 
         public void Dispose()
         {
+#if UNITY_STANDALONE || UNITY_EDITOR
+            if (DataStore.i.common.isWorldBeingDestroyed.Get())
+                return;
+#endif
+            
```

Comment:
What if we just put only one of these returns at the `catch` of `LoadScene()`? So it doesn't throw. 
Every Exception ends there.